### PR TITLE
Update the README now we have our first release

### PR DESCRIPTION
**Pull request by katheris**

### CODE REVIEW

File: packaging/helm-charts/helm3/strimzi-access-operator/README.md [hunk 1]
```diff
@@ -10,19 +10,60 @@ The operator is built using the [Java Operator SDK](https://github.com/java-oper
...
-
-To start the operator you need the Strimzi `Kafka` and `KafkaUser` custom resource definitions installed in your Kubernetes cluster.
-You can get these from the Strimzi [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator/tree/main/packaging/install/cluster-operator),
+For the operator to start successfully you need the Strimzi `Kafka` and `KafkaUser` custom resource definitions installed in your Kubernetes cluster.
+You can get these from the Strimzi [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator/tree/main/install/cluster-operator),
 or use the [Strimzi quickstart guide](https://strimzi.io/quickstarts/) to also deploy the Strimzi cluster operator and a Kafka instance at the same time.
 
+### Installing the Chart
+
+To install the chart with the release name `my-strimzi-drain-cleaner`:
```
Comment (Frawless) [inline packaging/helm-charts/helm3/strimzi-access-operator/README.md:19]:
```suggestion
To install the chart with the release name `my-strimzi-access-operator`:
```

Comment (katheris) [reply packaging/helm-charts/helm3/strimzi-access-operator/README.md]: Good spot!

**Review by ppatierno — APPROVED** [review]

**Review by see-quick — APPROVED** [review]
LGTM

**Review by Frawless — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -14,19 +14,39 @@ The operator is built using the [Java Operator SDK](https://github.com/java-oper
...
+The latest release of the Access Operator can be started using the manifests in the `install` directory.
+The [dev guide](https://github.com/strimzi/kafka-access-operator/blob/main/development-docs/DEV_GUIDE.md) describes how to build and run the Access Operator from source.
 
-To start the operator you need the Strimzi `Kafka` and `KafkaUser` custom resource definitions installed in your Kubernetes cluster.
-You can get these from the Strimzi [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator/tree/main/packaging/install/cluster-operator),
+For the operator to start successfully you need the Strimzi `Kafka` and `KafkaUser` custom resource definitions installed in your Kubernetes cluster.
+You can get these from the Strimzi [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator/tree/main/install/cluster-operator),
 or use the [Strimzi quickstart guide](https://strimzi.io/quickstarts/) to also deploy the Strimzi cluster operator and a Kafka instance at the same time.
 
+### Starting the Access Operator
```
Comment (scholzj) [inline README.md:24]: It is really deploying or installing and not starting it.

**Review by scholzj — CHANGES_REQUESTED** [review]

**Review by scholzj — APPROVED** [review]

**Pull request merged**